.cirrus.yml

```diff
@@ -98,3 +112,15 @@ task:
   environment:
     OS_NAME: freebsd
   << : *COMMON_STEPS_TEMPLATE
+
+task:
+  name: FreeBSD 11.4 x64, DMD (bootstrap)
```

Comment:
Why did you use a different FreeBSD version here?

Reply:
1) There's a huge difference between FreeBSD 11 and FreeBSD 12 (breaking ABI)
2) We currently test FreeBSD 11 on auto-tester

Reply:
Thanks for the explanation. Should this also test for `-m32` given that the auto-tester will be removed in the future?

Reply:
No, because we're about to deprecate the FreeBSD 32 binaries with the next release and for now we still have auto-tester coverage.

Reply:
Was not aware of that decision. Nevermind then

Reply:
It's not official yet, but I figured I rather wait before I invest time in it.